public/swagger.v1.json

```diff
@@ -36,6 +36,7 @@
         ],
         "summary": "Create a user",
         "operationId": "adminCreateUser",
+        "security": null,
```

Comment:
Why these changes?

Reply:
I've run `swagger generate spec` and this was the output. I didn't want to manually change it since it's easier to maintain this way.